tests/microos/workloads/firewalld-container/firewalld_client.pm

```diff
@@ -0,0 +1,48 @@
+# SUSE"s openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Package: podman firewalld-container
+# Summary: install and verify firewalld container.
+# Maintainer: QE Core <qe-core@suse.de>
+
+use base 'consoletest';
+use warnings;
+use strict;
+use testapi;
+use serial_terminal 'select_serial_terminal';
```

Comment:
When serial terminal would use that would be better.

Reply:
well spotted, just a leftover during development... Thanks